ztools/xssh/flag.go

```diff
@@ -90,19 +91,62 @@ func (kaList *KexAlgorithmsList) GetStringSlice() []string {
 	}
 }
 
+type CipherList struct {
+	Ciphers []string
+}
+
+func (cList *CipherList) String() string {
+	return strings.Join(cList.Ciphers, ",")
+}
+
+func (cList *CipherList) Set(value string) error {
+	for _, cipher := range strings.Split(value, ",") {
+		isValid := false
+		for knownCipher := range cipherModes {
+			if cipher == knownCipher {
+				isValid = true
+				break
+			}
+		}
+
+		if !isValid {
+			return errors.New(fmt.Sprintf("Can not support cipher : %s", cipher))
```

Comment:
Nits:

1. Error message should start with a lowercase letter
2. `Can` -> `do`
3. No space before the `:` (should be `cipher:`)